README.md

```diff
@@ -1598,10 +1598,1953 @@ end
+                   handleUpdate={this.handleUpdate} />
+      </div>
+    )
+  }
+```
+If everything is wired correctly, you should now see a select input with the four options in between the create skill form and the AllSkills list.
+<br>
+
+So now we will actually create our filtering functionality. The big picture is that we want to filter based on the level, so will start our journey by passing a filter property to our FilterSelect component.
+
+_body.js.jsx
+```
+  render() {
+    return (
+      <div className="container">
+        <NewSkill handleSubmit={this.handleSubmit} />
+        <FilterSelect filter={this.filterSkills} />
+        <AllSkills skills={this.state.skills}
+                   handleDelete={this.handleDelete}
+                   handleUpdate={this.handleUpdate} />
+      </div>
+    )
+  }
+```
+<br>
+
+I am sure you noticed that inside of our filter property we have a function called filterSkills, so let's go ahead and create that within our _body.js.jsx.
+
+```
+  filterSkills(value) {
+    console.log("about to filter");
+    debugger;
+  },
+```
+We will add the functionality of filtering in the above filterSkills(value) function. But first, let's wire it all together and make sure that when the user changes the select dropdown, this filterSkills function is invoked.
+<br>
+
+Let's add an onChange event to our select input that will call the updateFilter function. In order to accomplish that we must bind the object 'this' to our function call.
+
+_filter_selects.js.jsx
+```
+render (){
+    return(
+      <div>
+        <label>
+          Filter By
+        </label>
+        <select className="form-control" onChange={this.updateFilter.bind(this)}>
+        
+        ...
+```
+<br>
+Now we need to create an updateFilter function within the FilterSelect component.
+
+_filter_selects.js.jsx
+```
+updateFilter(event) {
+    this.props.filter(event.target.value)
+  }
+```
+Since we bound the object 'this' to the function call in our updateFilter, 'this' refers to the entire component. This gives us access to all of the properties of the FilterSelect component. We need the filter property, which we pass into the component when we render it in the Body. The filter property points to the filterSkills function in the Body component. Let's pass in the value of the select input, which we get from the event object.
+```
+event.target.value
+```
+
+If everything is working properly, when you reload the page and select an option from the selector, you should hit the debugger within the filterSkills function. You should also see "about to filter" logged to the console. From the console, you can type "value" and you will get the value you selected from the dropdown.
+<br>
+Next we need to actually make the filterSkills function filter our skills list based on the value passed in from the select input.
+
+_body.js.jsx
+```
+filterSkills(value) {
+    // Create a variable called skills set to an empty string
+    var skills = ''
```

Comment:
Also - this might just be me - but I find it incredibly difficult to read code that has explaining comments in it like below. I'd prefer if you explained and added code in a more iterative way (the way you would actually develop it) to make it easier to follow.